hack/local-up-cluster.sh

```diff
@@ -507,11 +511,8 @@ function start_apiserver {
       RUNTIME_CONFIG+="scheduling.k8s.io/v1alpha1=true"
     fi
 
-
-    # Admission Controllers to invoke prior to persisting objects in cluster
-    #
-    # The order defined here dose not matter.
-    ENABLE_ADMISSION_PLUGINS=LimitRanger,ServiceAccount${security_admission},DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,StorageObjectInUseProtection
```

Comment:
what about removing L106 and change this line to
```
ENABLE_ADMISSION_PLUGINS=${ENABLE_ADMISSION_PLUGINS:-LimitRanger,ServiceAccount${security_admission},DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,StorageObjectInUseProtection}
```

Reply:
I've removed the additional `DEFAULT_` variable but left (and updated) the list to match the recommended [defaults from the docs](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#is-there-a-recommended-set-of-admission-controllers-to-use)

It is my understanding that if --enable-admission-plugins is passed with an empty value/list, no plugins will be loaded.